app/api/users/route.ts

```diff
@@ -0,0 +1,10 @@
+import { PrismaClient } from '@prisma/client';
+import { NextResponse } from 'next/server';
+
+const prisma = new PrismaClient();
+
+export async function GET() {
+  const users = await prisma.user.findMany();
+  console.log(users);
```

Comment:
I would rather not console.log here

Reply:
Have removed all console logs in latest commit, waiting on fixes to build by @ludavidca before being able to pass deployments